src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write code here
+const allLi = document.querySelector('.tree').querySelectorAll('li');
+const tree = document.querySelector('.tree');
+
+for (const li of allLi) {
+  if (li.children.length !== 0) {
+    // <li> those haven't elements inside must be ignored
+    const span = document.createElement('span'); // we need just li those have.
+
+    // Create element span and insert it in every <li> we get in this condition
+    li.prepend(span); // span tag has a text like a 'neighbor' sibling
+    span.append(span.nextSibling); // move text inside in span
+  }
```

Comment:
This loop could be simplified. Instead of checking if `li.children.length` is not 0, you could directly check if `li` has any children using `if (li.hasChildNodes())`. This is more readable and performs better.